proto/interchain_security/ccv/provider/v1/tx.proto

```diff
@@ -7,11 +7,14 @@ import "google/api/annotations.proto";
 import "gogoproto/gogo.proto";
 import "cosmos_proto/cosmos.proto";
 import "google/protobuf/any.proto";
+import "ibc/lightclients/tendermint/v1/tendermint.proto";
+import "tendermint/types/evidence.proto";
```

Comment:
Is this import necessary?

Reply:
Good catch. The Tendermint `evidence` import isn't required here.